src/sass/claims-status.scss

```diff
@@ -265,22 +193,18 @@ a.claim-list-item {
 }
 
 .claims-status {
-  padding: 1em 0 3em 0;
+  padding: 1em 0 3em;
 }
 
-.your-claims, .claims-status {
+.claims-container, .claims-status {
   padding-bottom: 3em;
 
   h3 {
     margin-top: 1em;
   }
 
-  h1:first-child {
+  h1:first-child, .claims-container-title {
```

Comment:
When appeals status is cleaned, would it make sense to consolidate h1:first-child and claims-container-title as claims-container-title?

Reply:
That seems like it would make sense.